docs/monitoring.md

```diff
@@ -0,0 +1,35 @@
+# Monitoring
+
+This document describes how to get [Sentry](https://sentry.io/) (the Epic
+application monitoring provider) set up for error, performance, and replay
+monitoring.
+
+> **NOTE**: this is an optional step and only needed if you want monitoring in
+> production.
+
+## SaaS vs Self-Hosted
+
+Sentry offers both a [SaaS solution](https://sentry.io/) and
+[self-hosted solution](https://develop.sentry.dev/self-hosted/). This guide
+assumes you are using SaaS but the guide still works with self-hosted with a few
+modifications.
```

Comment:
I *love* this. Very much in the ethos of Epic Stack of avoiding vendor lock-in 👍 Thanks!

Reply:
@kentcdodds Yep! If you don't want to pay for Sentry, you can just run it yourself! Gotta love that open source mentality :D